cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -1,57 +1,51 @@
-{# ==========================================================================
+{# =========================================================================
 
    text_introduction.render()
 
-   ==========================================================================
+   =========================================================================
 
    Description:
 
    Create a Text Introduction molecule.
    See [GHE]/flapjack/Modules-V1/wiki/Text-Introduction
 
-   settings:           An object with the following options for settings.
+   settings:          An object with the following values for settings.
 
-   settings.heading:   Heading text.
-   settings.intro:     Body introduction text.
-   settings.body:      Body text.
+   settings.heading:  Heading text.
+   settings.body:     Body text.
 
-   settings.link_url:  Link URL.
-   settings.link_text: Link text.
+   options:           An object with the following values for options.
```

Comment:
Yep, just like I did on the hero. We don't have to organize them this way if everyone else disagrees, but it seems to be the most consistent to me.